mibs/dell/DHCP-MIB

```diff
@@ -0,0 +1,167 @@
+          DHCP-MIB DEFINITIONS ::= BEGIN
```

Comment:
Why is this in the dell folder ?

Reply:
Two options:
put them in a windows folder (new dir, no yaml changes needed)
put them in a microsoft folder and set `group: microsoft` in the definition yaml